colossalai/inference/struct.py

```diff
@@ -58,25 +58,35 @@ def __init__(
         self.request_id = request_id
         self.prompt = prompt
         self.input_token_id = token_id
-        self.blokc_size = block_size
+        self.block_size = block_size
         self.sample_params = sample_params
         self.output_token_id = []
         self.status = RequsetStatus.WAITING
         self.block_table_index = block_table_index
```

Comment:
the type of this attribute should be a 1d tensor for each sequence, not a int. To record the cache of a sequence like [addr_1, addr_2, ... ].

Reply:
Sure, fixed.